tools/actions/commit-queue.sh

```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
```

Comment:
I think it's fine for initial implementation though I'd prefer to have this in JS as this will allow for much bigger involvement and more robust code. Also, bash is easy to misuse and has lots of things that are easy to get wrong (quick googling https://mywiki.wooledge.org/BashPitfalls).
Though I'm guessing ncu doesn't have a proper API for this yet?

Reply:
Agreed. Main reason I didn't use JS is because ncu doesn't have an API, so writing in bash was just faster. Shouldn't be hard to rewrite it in JS though